save-cloud-common/src/commonMain/kotlin/org/cqfn/save/domain/Sdk.kt

```diff
@@ -0,0 +1,42 @@
+/**
+ * SDK which are supported for test execution in save-cloud
+ */
+
+package org.cqfn.save.domain
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name name of the SDK
+ * @property version
+ */
+@Serializable
+open class Sdk(val name: String, open val version: String) {
+    /**
+     * Should be used when no particular SDK is required
+     */
+    object Default : Sdk("ubuntu", "latest")
+
+    override fun toString() = "$name:$version"
+}
+
+/**
+ * @property version version of JDK
+ */
+class Jdk(override val version: String) : Sdk("openjdk", version)
+
+/**
+ * @property version version of Python
+ */
+class Python(override val version: String) : Sdk("python", version)
+
+/**
+ * Parse string to sdk
+ */
+fun String.toSdk() =
+        when (this) {
```

Comment:
I don't like such approach, it makes it too hard to support different version. Like, to support 10 different versions of Python you'll need 10 hardcoded lines here. I thought we will allow user to choose any valid tag for openjdk and python image